### ISSUE TITLE
Unable to run on Ubuntu 24.04.1

### ISSUE THREAD

**Issue by sebastianschauenburg** [muts/qbr#42]
So, I tried to get this amazing tool running on Ubuntu 24.04.1, but that did not go as planned. Mostly Python issues (my current version is Python 3.12.3).  
- According to [this thread on StackOverflow](https://stackoverflow.com/questions/77247893/modulenotfounderror-no-module-named-distutils-in-python-3-12) Python 3.12 removed ```distutils```. So I of course tried setuptools.
- Eventually I even tried a distrobox Debian 11 image, to get it installed, but then I could not run it due to not being able to access the hardware.
- Apologies of the messy issue report, I had many papercuts. 

Concrete questions:
- could you perhaps provide some documentation on what system you are running this on, with which versions of Python etc.? 
- do you plan on upgrading this project to Python 3.12 and onward?

It sounds like an amazing project, thank you for all your efforts :+1:

**Comment by kkoomen** [comment]
> could you perhaps provide some documentation on what system you are running this on, with which versions of Python etc.?
Not really, the README is up-to-date with what I developed at the time. If you experience problems, then you can debug this yourself, because this project is done for me and I have no intentions for maintaining it. I only merge PRs and that's it.

> do you plan on upgrading this project to Python 3.12 and onward?
Nope, but feel free to submit a PR with fixes that are required.
